tests/test_opening_hours.py

```diff
@@ -99,6 +99,11 @@ def test_multiple_times():
     o.add_range("Mo", "13:00", "17:30")
     assert o.as_opening_hours() == "Mo 08:00-12:00,13:00-17:30"
 
+    o2 = OpeningHours()
+    o2.add_range("Tu", "09:00", "12:00")
+    o2.add_range("Tu", "15:00", "17:00")
```

Comment:
Should the test be the other way?
```suggestion
    o2.add_range("Tu", "15:00", "17:00")
    o2.add_range("Tu", "09:00", "12:00")
```

Reply:
Added another test case. The `o2` case is interesting because it fails with the current main branch, ie. without the `sorted()` call that this pull request adds to the opening hours implementation.